.github/workflows/build-linux.yml

```diff
@@ -0,0 +1,77 @@
+name: Build(Linux packages)
+
+on:
+  push:
+    branches:
+      - master
+      - refactor-b # temp
+    tags:
+      - v*
+  pull_request:
+
+jobs:
+  build-linux:
+    name: Build (Unix-like OSes)
+    uses: ./.github/workflows/go-build.yml
+    with:
+      os-version: ubuntu-latest
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y devscripts debhelper fakeroot binutils-mips-linux-gnu binutils-aarch64-linux-gnu binutils-arm-linux-gnueabihf
+        docker pull mackerel/docker-mackerel-rpm-builder:c7
+        make rpm deb rpm-kcps deb-kcps rpm-stage deb-stage tgz
+        make crossbuild
+      upload-artifact-path: |
+        rpmbuild/RPMS/*/*.rpm
+        packaging/*.deb
+        snapshot/*.zip
+        snapshot/*.tar.gz
+        build/*.tar.gz
+  build-windows-x64:
```

Comment:
The job `build-windows-x64` and `build-windows-x86` are not seems to be `build-linux.yml`.